octoprint_mrbeam/__init__.py

```diff
@@ -1918,6 +1926,8 @@ def on_api_command(self, command, data):
             return NO_CONTENT
         elif command == "focus_reminder":
             return self.focus_reminder(data)
+        elif command == "laserhead_changed":
+            return self.laserhead_changed(data)
```

Comment:
... so, in the backend, you can receive the user acknowledgment, and then decide to 
```py
self._settings.set_boolean(["laserheadChanged"], dict_get(data, "user_acknowledged", False)
```
I do not see the need for a separate `laserhead_changed()` function.

Reply:
I'm not sure what you mean here

Reply:
basically it was shortening it too

Reply:
nvm